arch/x86/include/kernel_arch_data.h

```diff
@@ -45,6 +45,8 @@
 #define DATA_SEG	0x10
 #define MAIN_TSS	0x18
 #define DF_TSS		0x20
+#define USER_CODE_SEG	0x2b /* at dpl=3 */
+#define USER_DATA_SEG	0x33 /* at dpl=3 */
```

Comment:
Just curious: why these particular descriptor numbers?  The others are all multiples of 8, but these look funny.

Reply:
the two low bits are the privilege level, which is why 0x3 is ANDed in.